src/coreclr/vm/dllimport.h

```diff
@@ -203,6 +203,9 @@ enum ILStubTypes
     ILSTUB_TAILCALL_CALLTARGET           = 0x80000009,
     ILSTUB_STATIC_VIRTUAL_DISPATCH_STUB  = 0x8000000A,
     ILSTUB_DELEGATE_INVOKE_METHOD        = 0x8000000B,
+#ifdef TARGET_RISCV64
+    ILSTUB_DELEGATE_SHUFFLE_THUNK        = 0x8000000C,
+#endif
```

Comment:
```suggestion
    ILSTUB_DELEGATE_SHUFFLE_THUNK        = 0x8000000C,
```
It is fine to define these constants without an ifdef. (Some of the above constants are arch-specific as well and they do not have an ifdef.)